site-packages/integralstor_utils/zfs.py

```diff
@@ -1634,6 +1634,56 @@ def _get_size_components(st):
         return d, None
 
 
+def to_bytes(sizes_list):
+    """Converts to bytes
+
+    args:        each entry in sizes_list must be of format 'value unit' (100 M)
+    returns:     a list with entries in bytes, no suffix attached.
+    """
+
+    in_bytes = []
+    try:
+        defs = {'KB': 1024, 'MB': 1024**2, 'GB': 1024**3, 'TB': 1024 **
+                4, 'K': 1024, 'M': 1024**2, 'G': 1024**3, 'T': 1024**4}
+        in_bytes = [float(val) * defs[unit]
+                    for val, unit in [l.split() for l in sizes_list]]
+
+    except Exception, e:
+        return None, str(e)
+    else:
+        return in_bytes, None
+
+
+def bytes_to_human_readable_size(in_bytes):
```

Comment:
On a second thought, I'll not use the function get_naturalsize() in filesize.py since it doesn't return any error strings. To make things worse, it has multiple return statements. Instead of fixing that, I'll use the new function 'bytes_to_human_readable_size()' since I've already spent some time on it.